operator/internal/controller/vectorized/cluster_controller_configuration_drift_test.go

```diff
@@ -17,6 +17,28 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestDiffWithNull(t *testing.T) {
```

Comment:
Would you mind copying the setup of a similar test that we have in `operator/cmd/syncclusterconfig`? We stand up a redpanda test container to make sure that the config is actually accepted by the redpanda instance. Way faster than a full k8s setup with much more validation (and no need to write out the ConfigPropertyMetadata manually)

Reply:
Yep will add more test, i am interested in a few corner cases